lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -227,19 +228,25 @@ sub create_from_settings {
     return $job;
 }
 
-sub rg_files ($self, $dir, $module_re) {
-    my $res = run_cmd_with_log_return_error(['rg', '--type','perl', '--type', 'py', '-l', $module_re, $dir]);
-    return [] unless my $files = $res->{stderr};
-    return [map { $_ =~ s/\..*$//; basename $_ } split(/\n/, $files)];
+sub search_modules ($self, $module_re) {
+    my $distris = path(testcasedir);
+    my @results;
+    for my $distri ($distris->list({dir => 1})->map('realpath')->uniq()->each) {
```

Comment:
this looks neat :)

Reply:
> Sorry, I don't understand the git commit message "git grep based on chris search_modules in api".
> The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.
> 
I't was temporary commit message
> Please also explain why you add the function "run_cmd_with_log_return_all"

Wanted function which returns correctly stdout and stderr ( not merged ), and I dont like in place solution like in https://github.com/os-autoinst/openQA/blob/7d05a925004d7ec4738568a06c786aebdd1f4bb0/lib/OpenQA/WebAPI/Controller/API/V1/Search.pm#L73-L96